### ISSUE TITLE
Publish an Ivy distribution

### ISSUE THREAD

**Issue by mraible** [okta/okta-angular#86]
## I'm submitting a:

- [ ] Bug report  
- [x] Feature request
- [ ] Other (Describe below)

## Current behavior

When you build an Angular 13 app with the Okta SDK, it advises you to publish an Ivy distribution.

```
Processing legacy "View Engine" libraries:                                
  - @okta/okta-angular [es2015/esm2015] (https://github.com/okta/okta-angular)                                                                       
 Encourage the library authors to publish an Ivy distribution.
```

## Expected behavior

There are no warnings when using Angular 13.

## Minimal reproduction of the problem with instructions

Create an Angular 13 app, add Okta, run `npm build`.

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for the report @mraible.

Internal ref: OKTA-469350

**Comment by trie-me** [comment]
Still present in Angular 14, more warnings:

```
Warning: /home/REDACTED/src/REDACTED/node_modules/@okta/okta-auth-js/esm/esm.browser.js depends on '@babel/runtime-corejs3/regenerator'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/REDACTED/src/REDACTED/node_modules/@okta/okta-auth-js/esm/esm.browser.js depends on 'cross-fetch'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/REDACTED/src/REDACTED/node_modules/@okta/okta-auth-js/esm/esm.browser.js depends on 'p-cancelable'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/REDACTED/src/REDACTED/node_modules/@okta/okta-auth-js/esm/esm.browser.js depends on 'tiny-emitter'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/REDACTED/src/REDACTED/src/app/auth/login/login.component.ts depends on '@okta/okta-signin-widget'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```
Also of note, the auth chunk with okta sign in widget is larger than the rest of the application, can't be optimized or tree shaken:

```
Initial Chunk Files           | Names                                          |  Raw Size | Estimated Transfer Size
main.c821ca1cde047ad3.js      | main                                           | 663.38 kB |               168.97 kB
styles.e578d4cc2b33eb98.css   | styles                                         |  96.04 kB |                11.28 kB
polyfills.f17026223e421b39.js | polyfills                                      |  33.07 kB |                10.65 kB
runtime.589529829420e722.js   | runtime                                        |   2.91 kB |                 1.42 kB

                              | Initial Total                                  | 795.39 kB |               192.31 kB

Lazy Chunk Files              | Names                                          |  Raw Size | Estimated Transfer Size
459.46474bf25f15bd73.js       | auth-login-login-module                        |   1.40 MB |               282.14 kB
```

**Comment by samartomar** [comment]
Please help us with updated lib. this is the only lib now still not in IVY. 

- @okta/okta-angular [es2015/esm2015] (https://github.com/okta/okta-angular)
DeprecationWarning: 'getMutableClone' has been deprecated since v4.0.0. Use an appropriate `factory.update...` method instead, use `setCommentRange` or `setSourceMapRange`, and avoid setting `parent`.
Encourage the library authors to publish an Ivy distribution.
DeprecationWarning: 'getMutableClone' has been deprecated since v4.0.0. Use an appropriate `factory.update...` method instead, use `setCommentRange` or `setSourceMapRange`, and avoid setting `parent`.

**Comment by denysoblohin-okta** [comment]
@samartomar What exact Angular version do you use and typescript version?

**Comment by denysoblohin-okta** [comment]
Ivy distribution is available since version 6.0.0
